e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -225,21 +225,123 @@ test.describe('Example Imagery', () => {
+  await bgImageLocator.hover();
+
+  // Click previous image button
+  const previousImageButton = await page.locator('.c-nav--prev');
+  await previousImageButton.click();
+
+  // Verify previous image 
+  const selectedImage = page.locator('.selected');
+  await expect(selectedImage).toBeVisible();
+
+  // Zoom in
+  const originalImageDimensions = await page.locator(backgroundImageSelector).boundingBox();
+  await bgImageLocator.hover();
+  const deltaYStep = 100; // equivalent to 1x zoom
+  await page.mouse.wheel(0, deltaYStep * 2);
+  const zoomedBoundingBox = await bgImageLocator.boundingBox();
+  const imageCenterX = zoomedBoundingBox.x + zoomedBoundingBox.width / 2;
+  const imageCenterY = zoomedBoundingBox.y + zoomedBoundingBox.height / 2;
+
+  // Wait for zoom animation to finish 
+  await bgImageLocator.hover();
+  const imageMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+  expect(imageMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+  expect(imageMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+  // Center the mouse pointer
+  await page.mouse.move(imageCenterX, imageCenterY);
+
+  // Pan Imagery Hints
+  console.log('process.platform is '+process.platform);
+  const expectedAltText = process.platform === 'linux' ? 'Ctrl+Alt drag to pan' : 'Alt drag to pan';
+  const imageryHintsText = await page.locator('.c-imagery__hints').innerText();
+  expect(expectedAltText).toEqual(imageryHintsText);
+
+  // Click next image button
+  const nextImageButton = await page.locator('.c-nav--next');
+  await nextImageButton.click();
+
+  // Click fixed timespan button
+  await page.locator('.c-button__label >> text=Fixed Timespan').click();
+
+  // Click local clock 
+  await page.locator('.icon-clock >> text=Local Clock').click();
+
+  // Zoom in on next image
+  await bgImageLocator.hover();
+  await page.mouse.wheel(0, deltaYStep * 2);
+
+  // Wait for zoom animation to finish
+  await bgImageLocator.hover();
+  const imageNextMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+  expect(imageNextMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+  expect(imageNextMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+   // Click previous image button
+   await previousImageButton.click();
+
+   // Verify previous image
+   await expect(selectedImage).toBeVisible();
+
+  // Wait 20ms to verify no new image has come in
+  await page.waitForTimeout(21);
+
+  //Get background-image url from background-image css prop
+  const backgroundImage = await page.locator('.c-imagery__main-image__background-image');
+  let backgroundImageUrl = await backgroundImage.evaluate((el) => {
+  return window.getComputedStyle(el).getPropertyValue('background-image').match(/url\(([^)]+)\)/)[1];
+  });
+  let backgroundImageUrl1 = backgroundImageUrl.slice(1, -1); //forgive me, padre
+  console.log(backgroundImageUrl1);
```

Comment:
when using console.log, please print the variable you're logging to console like

```js
console.log('backgroundImageUrl1 ' + backgroundImageUrl1)
```